exchanges/orderbook/orderbook_types.go

```diff
@@ -38,23 +38,23 @@ var (
 	errChecksumStringNotSet = errors.New("checksum string not set")
 )
 
-var service = Service{
-	books: make(map[string]Exchange),
-	Mux:   dispatch.GetNewMux(nil),
+var s = store{
+	orderbooks:  make(map[key.ExchangePairAsset]bookWithExchangeID),
+	exchangeIDs: make(map[string]uuid.UUID),
+	signalMux:   dispatch.GetNewMux(nil),
 }
 
-// Service provides a store for difference exchange orderbooks
-type Service struct {
-	books map[string]Exchange
-	*dispatch.Mux
-	mu sync.Mutex
+type bookWithExchangeID struct {
```

Comment:
🐒 Think this is just a book 🤷 
```suggestion
type book struct {
```